guide/faq/authenicated-access-to-extra-content.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Mission Portal authenticated access to extra content
```

Comment:
I think I would rename this to Extending Mission Portal and probably I would put it under https://docs.cfengine.com/docs/3.12/enterprise-cfengine-guide-hub_administration.html instead of the main FAQ section because it is enterprise specific and does not apply to core at all.